### ISSUE TITLE
Advanced Web Interface - Move Discover, Personal and Trending to left most pane.

### ISSUE THREAD

**Issue by TheDukeDK** [mastodon/mastodon#20096]
### Pitch

The advanced web interface is banging! But a bit of screen space could be saved by moving the contents of the Getting Started pain to left most pane and being able to scroll the pane up and down. 

![image](https://user-images.githubusercontent.com/8044990/200491595-37ea68d0-0ba0-430a-b412-8552bea7b042.png)

### Motivation

I think it is just a waste of screen space to have that info always on the right. :shrug:

**Comment by cgillogly** [comment]
I agree. Combining it with the left most panel would save space and also stop the need to scroll back and forth when switching between the different feeds / searches. When searching and clicking on someone, I find it annoying that it scrolls to the right and then I have to scroll back to get where I was.

If combining them it's an option, the next best thing would be to allow it to be moved to a specific spot, just like you can with other columns so it can be put next to the first column instead.

**Comment by stibinator** [comment]
I wrote a userscript that does it on the client side, if anyone's interested https://greasyfork.org/en/scripts/468037-rearrange-o-don

**Comment by jpluimers** [comment]
> I wrote a userscript that does it on the client side, if anyone's interested https://greasyfork.org/en/scripts/468037-rearrange-o-don

Is there any place where I can post issues on your rearrange-O-Don?

Currently it hides search results.

For the rest: great script (especially when combined with your resize-o-don at https://greasyfork.org/en/scripts/454111-resize-o-don, for the bug it does not matter if both are running at the same time: having only enabled rearrange-O-Don is enough)

Searching for "stevetex" with rearrange-O-Don disabled:

![image](https://github.com/mastodon/mastodon/assets/2033367/d85afe34-833e-48ec-9d0f-28952981057a)

Same but with rearrange-O-Don enabled:

![image](https://github.com/mastodon/mastodon/assets/2033367/59caa7cc-9021-4c54-ab20-9687021fe27a)

**Comment by stibinator** [comment]
That's a known bug. Its status is Can't Fix (I don't know enough to be able to implement a fix for it, hopefully someone with some actual web design skills forks and fixes it). I don't do a lot of searching.

**Comment by jpluimers** [comment]
> That's a known bug. Its status is Can't Fix (I don't know enough to be able to implement a fix for it, hopefully someone with some actual web design skills forks and fixes it). I don't do a lot of searching.

Thanks. Is there a status page for the issue I can watch (in case someone else mages progress), or - when energy levels further increase, see [Twitter profile](https://twitter.com/jpluimers) - I can contribute to?

**Comment by stibinator** [comment]
Not yet, I'll have to look up how to do that

**Comment by jpluimers** [comment]
> Not yet, I'll have to look up how to do that

Thanks. Please post progress here. No hurry though: life is more than doing GitHub stuff (;
